doc/connectivity/region_extraction.rst

```diff
@@ -31,7 +31,7 @@ dataset.
 
 .. literalinclude:: ../../examples/03_connectivity/plot_extract_regions_dictlearning_maps.py
     :start-after: # We use nilearn's datasets downloading utilities
-    :end-before: ################################################################################
+    :end-before: ##############################################################################
```

Comment:
This is totally my doing. Sorry about that.

For my defense, I think that those separators have different length in different examples files. Is that intended?

Reply:
Hard to say but it's not a big deal I think and an easy fix!